wasp/boards/simulator/watch.py

```diff
@@ -22,6 +23,70 @@ def print_exception(exc, file=sys.stdout):
 from drivers.st7789 import ST7789_SPI
 from drivers.vibrator import Vibrator
 
+class simCST816S(object):
+    """Simulate touch screen function on simulator.
+    This class for simulate touch screen driver to match used in wasp.py
+    because the simulator doesn't support interrupt handler    
+    """
```

Comment:
I'm afraid I don't much like this. There's no point in having the simulator emulate a CST816S if we don't use the real driver to simulate it. Running the real drivers on the simulator makes testing for silly errors (like wrong variable names) so much quicker than having to reflash the target device.

Adding interrupt suppport to the simulator should be pretty easy. I think it's just a matter of adding irq() and raise() calls to the Pin object).